test/helpers/kubectl.go

```diff
@@ -1215,6 +1215,14 @@ func (kub *Kubectl) GatherLogs() {
 		return
 	}
 	reportMap(testPath, reportCmds, kub.SSHMeta)
+
```

Comment:
Why call `reportMap` twice here (on line 1217 and on line 1224) instead of just appending the commands for getting kubelet logs to `reportCmds` that is used in the first call to `reportMap`?

Reply:
because the sshMeta is different.

Reply:
Ah, got it!